Jhur008 led

jhur008 opened the pull request:
Software for LED belt. Has appropriate danger levels. Ready for server integration.

jhur008 committed:
Edited buzzer and servo into a sprayer file that uses both.

jhur008 committed:
Merge branch 'jhur008_LED' of https://github.com/ucr-cs179j-team02/CS179J-Team02 into jhur008_LED

jhur008 committed:
Basic LED parallel lightup

jhur008 committed:
Added temp danger levels and defined them.

jhur008 committed:
Completed danger levels. Program is ready for server integration.

edick007 reviewed (changes requested):
For the LED belt We are not using regular LEDS, we are using ws2812B leds. These can change any color and have only one pin needed to control all of them. Use the fast LED library to make things easier. Here is a tutorial. https://howtomechatronics.com/tutorials/arduino/how-to-control-ws2812b-individually-addressable-leds-using-arduino/

jhur008 committed:
LED danger levels for ws2812B LEDs

edick007 reviewed (approved):
Looks good will upload to the belt and submit feedback as it comes!

edick007 committed:
Update LED.ino

Changes settings and some code to have the proper intended animations and configuration
